README.md

```diff
@@ -3,18 +3,18 @@ An approach to make code "production-ready"
 
 There is no "one true way" to write code or make it production-ready. What _production-ready_ means will highly depend on what the code does and the context in which you will use it. However, it is generally useful to try and prioritise among all the things that you could do to increase the level of confidence that you have in your code, and approach that task in a systematic way.
 
-> The general rule of thumb is: **don't do more than needed**. The sooner your code is in production, the sooner it actually starts being useful. You are only judge of what is _needed_, and it is a difficult question to answer. Different people around you will have different thoughts about what that means. Try to stay open-minded about it, accept that you might not be fully comfortable with the code you ship to production, but be reasonable, think through the consequences and don't put yourself in a position where you'd be tempted to deny responsibility for what you ship — because you _are_ ultimately responsible for what the code you write does.
->
-> I'll do my best to help. Now take a breath, it takes practice, but you can do it! ; ) — [GB](https://github.com/gonzalo-bulnes)
+The general rule of thumb is: **don't do more than needed**. The sooner your code is in production, the sooner it actually starts being useful. You are only judge of what is _needed_, and it is a difficult question to answer. Different people around you will have different thoughts about what that means. Try to stay open-minded about it, accept that you might not be fully comfortable with the code you ship to production, but be reasonable, think through the consequences and don't put yourself in a position where you'd be tempted to deny responsibility for what you ship — because you _are_ ultimately responsible for what the code you write does.
+
+I'll do my best to help. Now take a breath, it takes practice, but you can do it! ;)
```

Comment:
```suggestion
> I'll do my best to help. Now take a breath, it takes practice, but you can do it! ; ) — [GB](https://github.com/gonzalo-bulnes)
```

To go with the suggestions on the lines above.
Despite not being --strictly speaking-- quotes, I find these highlighted blocks benefit from having the "source"/personal touch at the end, that's why I'd keep the initials.

This has the unhappy link effect you noticed when reading in rendered form locally, but I'd consider it enough of an edge case to still keep this. (In part because even when reading locally, the link is clear in plain text, and the potential for confusion only comes with the combination of local + HTML render, which is not to say it doesn't matter, but I think is an acceptable imperfection :stuck_out_tongue:.)

Reply:
Also to some extent I find that keeping the "signature" allows to skip the "AUTHOR'S" part in "AUTHOR'S NOTE" :stuck_out_tongue:

Reply:
The initials are perfectly valid when is does not look like an actual quote. Otherwise you are just recursively quoting yourself 💥 
So yes, solved! :D